src/components/MarkdownDisplay.tsx

```diff
@@ -15,11 +17,24 @@ interface Props {
 }
 
 const MarkdownDisplay: React.FC<Props> = ({ treeCore }): React.ReactElement => {
-  const [markDownTree, setMarkDownTree] = useState<string[]>(
-    generateMarkDownTree(treeCore)
-  );
+  const [filter, setFilter] = useState<string | null>(null);
+  const filtering = (filter): Function | null => {
+    if (filter === "ROOT_ONLY") {
+      return selectRootCores;
+    } else {
+      return null;
+    }
+  };
+
+  useLayoutEffect(() => {
```

Comment:
Why not useEffect??

Reply:
Yes... I was trying something different, I forgot to put it back to useEffect